turbopack/crates/turbopack-ecmascript-plugins/src/transform/swc_ecma_transform_plugins.rs

```diff
@@ -186,7 +186,7 @@ impl CustomTransformer for SwcEcmaTransformPluginsTransformer {
                         PluginSerializedBytes::try_serialize(&module_program)?;
 
                     let transform_metadata_context = Arc::new(TransformPluginMetadataContext::new(
-                        Some(ctx.file_name_str.to_string()),
+                        Some(ctx.file_path_str.to_string()),
```

Comment:
It gives a path like 
```
filename: Some("src/app/page.tsx")
filename: Some("node_modules/.pnpm/file+..+next-turbopack+packages+next_react-dom@19.1.0_react@19.1.0/node_modules/next/dist/esm/build/templates/helpers.js")
```

Reply:
would it make sense to make the webpack paths relative too so plugins behave the same?

Reply:
I think so. I'll try to fix it soon.